### ISSUE TITLE
Generated PDF does not include text

### ISSUE THREAD

**Issue by l4b4r4b4b4** [nextcloud/workflow_pdf_converter#207]
## Steps to reproduce
1. Setup Nextcloud with Docker-Compose
2. Install OpenJDK8 & LibreOffice in Nextcloud Docker-Container
3. Setup Nextcloud Cron with Docker
3. Define Flow-Rule for automated PDF-File creation for Documents

### Expected behaviour
Generation of PDF-file with document's content.

### Actual behaviour
The generated PDF-file does not include text. All characters are replaced by generic squares.  
[Uploading Umwandeln (1).pdf…]()

## PDF Converter app

**PDF Converter app version:** 1.5.0

## Server configuration
**Operating system**: Docker on Debian

**Web server:** Nginx

**Database:** Maria

**PHP version:** 7.4

**Nextcloud Version:** 20.0.1

**Where did you install Nextcloud from:**
Docker Hub

**Are you using encryption:** no

**Comment by l4b4r4b4b4** [comment]
Had to install more more than just libreoffice-core. The following Code in the NC Dockerfile resolved the issue!
```
RUN apk add --no-cache \
    dumb-init \
    openjdk8 \
    libreoffice \
    libreoffice-base \
    libreoffice-lang-de \
    mesa-gl \
    ttf-freefont
```

**Comment by Manuel-Haas** [comment]
This issue is still relevant for Nextcloud AIO (which is the recomended installation method). I've tried to add the above APKs to the container, using the official method as [mentioned in NC Docs](https://github.com/nextcloud/all-in-one?tab=readme-ov-file#how-to-add-os-packages-permanently-to-the-nextcloud-container)).

Libreoffice seems to be installed fine, as I can call it within the container nextcloud-aio-nextcloud:
```
117d633edaf0:/var/www/html# which libreoffice
/usr/bin/libreoffice
117d633edaf0:/var/www/html# libreoffice --version
LibreOffice 7.6.7.2 60(Build:2)

117d633edaf0:/var/www/html# 
```
But the PDF is stll not updated, and in the container logs I find this message:
`Fontconfig error: No writable cache directories`

… which has already been [mentioned here](https://help.nextcloud.com/t/nextcloud-aio-workflow-only-numbers-in-pdf/157386) - but sadly the issue has also been closed a year ago.
